src/test/java/com/artipie/asto/LoggingStorageTest.java

```diff
@@ -102,4 +115,45 @@ void saveOverwrites() {
             new IsEqual<>(updated)
         );
     }
+
+    @Test
+    void logsWhenListingKeys() {
+        this.memsto.save(
+            new Key.From("one/two"),
+            new Content.From("data1".getBytes(StandardCharsets.UTF_8))
+        ).join();
+        this.memsto.save(
+            new Key.From("one/three"),
+            new Content.From("data2".getBytes(StandardCharsets.UTF_8))
```

Comment:
as content of the storage item does not matter here, `Content.EMPTY` can be used

Reply:
@olenagerasimova You are right !